items/swords/Voidforge.yml

```diff
@@ -0,0 +1,18 @@
+acquisitionMethods:
+  - type: plainText
+    text: "'The Endless Darkness' divination cards can be passively farmed while you target-farm Harbinger uniques from the boss encounter in T15 Beachhead maps."
+    referenceLink: https://poe.fish/fishing-items/anglers-plait
```

Comment:
This one is possibly redundant with the `typicalDivCardFarm` source listed below (and has an unrelated ref. link) - consider revising or keeping only one of the two

Reply:
Woops. Totally missed the link!